docs/examples/otlp/README.md

```diff
@@ -0,0 +1,44 @@
+## What
```

Comment:
Even if I understand why you did that, can you remove the whole example folder please. I prefer we all think about what we want to do before integrating so much new things in the repo. Especially it's not a good idea to embed falco rules there.

Reply:
Done at 289f2f06.